build/tools/projects.json

```diff
@@ -0,0 +1,74 @@
+{
+  "tasks": {
+    "setup": {
+      "description": "install and build all project dependencies",
+      "command": "cd {fullpath} {perDependencyCommand} && npm ln && npm run build",
+      "perDependencyCommand": " && npm ln {dependency}"
+    },
+    "teardown": {
+      "description": "un-install all project dependencies",
```

Comment:
>un-install [](start = 22, length = 10)

nit: I believe "uninstall" without a hyphen is ok. I'd also add "deregister the package from the cache"